network/local.go

```diff
@@ -199,25 +201,27 @@ func NewLocalConnWithManager(ctx *LocalManager, local, remote Address) (*LocalCo
 
 // Send implements the Conn interface.
 func (lc *LocalConn) Send(ctx context.Context, msg Body) error {
-
-	var body Body
-	var val = reflect.ValueOf(msg)
-	if val.Kind() == reflect.Ptr {
-		val = val.Elem()
-	}
-	body = val.Interface()
-
-	var typ = TypeFromData(body)
-	nm := Packet{
-		MsgType: typ,
-		Msg:     body,
+	buff, err := MarshalRegisteredType(msg)
```

Comment:
I'm not really a fan but it is because we want to have a working `Tx()` and `Rx()` method so I got to marshal the message now ..  (before it was not needed to marshal).